das-cli/src/common/bus_node/busnode_command_registry.py

```
@@ -50,58 +48,63 @@ def _gen_default_cmd(self, service, endpoint, ports_range):
+        return self._gen_default_cmd(service, endpoint, ports_range)
+
+    def cmd_query_engine(self, service, endpoint, ports_range, options, **args):
         base = self._gen_default_cmd(service, endpoint, ports_range)
         attention_broker = (
             f"{options['attention_broker_hostname']}:{options['attention_broker_port']}"
         )
 
         return f"{base} --attention-broker-endpoint={attention_broker}"
 
-    def _cmd_evolution_agent(self, service, endpoint, ports_range, options, **args):
+    def cmd_evolution_agent(self, service, endpoint, ports_range, options, **args):
```

Ruff (0.16.1)

[warning] Missing type annotation for `**args`

(ANN003)